testing_scripts/ScriptInfo.txt

```diff
@@ -0,0 +1,81 @@
+
+INFORMATION ON LFIND TESTING SCRIPT 
+(there will probably be bugs or errors running on other peoples systems, so please let me know what those errors are so they can be fixed)
+
+<><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><>
+To use script similar to older script:
+    1) You need to have a folder or collection of folders that only include coq files (not _CoqProject or Makefiles or anything else)
```

Comment:
I had just thought it would be easier to not have the user/tester have to create the _CoqProject file and such; I guess my thought process was that if the benchmarks are coq files then we would want each test to just take in those files and the handle everything for the user. I can change it to check if a make file and coqproject file exist and then use those as opposed to making them if that is preferred, that wouldn't be difficult just a few more lines of code.